org.natron.Natron.json

```diff
@@ -0,0 +1,841 @@
+            "strip-components": 0
+        },
+        {
+            "type": "patch",
+            "path": "patches/qt-everywhere-opensource-src-4.8.7-gcc6.patch"
+        },
+        {
+            "type": "patch",
+            "path": "patches/qt4-kde-fix.diff",
+            "strip-components": 0
+        },
+        {
+            "type": "patch",
+            "path": "patches/qt4-pcre.patch",
+            "strip-components": 0
+        },
+        {
+            "type": "patch",
+            "path": "patches/qt4-selection-flags-static_cast.patch"
+        },
+        {
+            "type": "patch",
+            "path": "patches/patch-src_corelib_io_qprocess_unix.cpp.diff",
+            "strip-components": 0
+        }
+    ],
+    "cleanup": [
+            "/bin",
+            "/include",
+            "/lib/*.a",
+            "/lib/*.la",
+            "/lib/*.prl",
+            "/lib/pkgconfig",
+            "/lib/qt4/mkspecs",
+            "/lib/qt4/q3porting.xml"
+        ]
+    },
+    {
+            "name": "boost",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap.sh --prefix=/app --with-libraries=date_time,filesystem,iostreams,python,serialization,locale,regex,system,thread",
+                "./b2 --build-type=minimal link=shared",
+                "./b2 --build-type=minimal link=shared install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://sourceforge.net/projects/boost/files/boost/1.66.0/boost_1_66_0.tar.bz2",
+                    "sha256": "5721818253e6a0989583192f96782c4a98eb6204965316df9f5ad75819225ca9"
+                }
+            ],
+            "cleanup": [
+                "/include"
+            ]
+        },
+        {
+            "name": "tinyxml",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://sourceforge.net/projects/tinyxml/files/tinyxml/2.6.2/tinyxml_2_6_2.tar.gz",
+                    "sha256": "15bdfdcec58a7da30adc87ac2b078e4417dbe5392f3afb719f9ba6d062645593"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/tinyxml_CMakeLists.patch"
+                }
+            ],
+            "cleanup": [
+                "/include"
+            ]
+        },
+        {
+            "name": "yaml-cpp",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_POSITION_INDEPENDENT_CODE=ON"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/jbeder/yaml-cpp/archive/yaml-cpp-0.5.3.tar.gz",
```

Comment:
I think you can use the most recent yaml-cpp if using a C++11 compiler and the latest OpenColorIO